package.json

```diff
@@ -66,8 +66,6 @@
     "react-native-progress": "^3.4.0",
     "react-native-push-notification": "^3.1.1",
     "react-native-scalable-image": "^0.4.0",
-    "react-native-svg": "6.3.1",
-    "react-native-svg-uri": "^1.2.3",
```

Comment:
Great to get rid of these, but we should remove all the native deps and code from the Xcode and Android projects too.

Reply:
Hmm, also not sure if SB is counting on SVG being there for some of the animations they made...

Reply:
Do whatever you think is right here, then merge away